niceday-api/.eslintrc.yml

```diff
@@ -8,3 +8,5 @@ extends:
 parserOptions:
   ecmaVersion: 12
 rules: {}
+globals:
+  fetch: false
```

Comment:
What does this do? (Maybe document it or else I just want to know)

Reply:
Maybe a true JS expert would know how to make JS a non-disgusting language, but I'm afraid it's not in my power. `require('isomorphic-fetch');` silently adds the `fetch` function as a global, so eslint then chokes on it because it's not "defined" anywhere in our code. I tried importing it explicitly by name but nothing I tried worked. All the answers online assume you just use it as we do, anyway, and that you tell eslint to ignore that global. I've added a brief comment to the config file to remind us why it's there later.